src/lib/sync/pr_info.ts

```
@@ -36,20 +69,18 @@ export async function syncPRInfoForBranches(
     API_SERVER,
     graphiteCLIRoutes.pullRequestInfo,
     {
-      authToken: authToken,
-      repoName: context.repoConfig.getRepoName(),
-      repoOwner: context.repoConfig.getRepoOwner(),
+      ...params,
       prNumbers: [...existingPrInfo.keys()],
       // For branches that are not already associated with a PR, fetch by branch name.
       prHeadRefNames: [...branchesWithoutPrInfo],
     }
   );
 
   if (response._response.status !== 200) {
-    return;
+    return [];
```

should we log splog.debug message here or something? Or perhaps that should be the job of the the API generic request function

---

yeah, thats what i'm thinking.  for now we can fail silently for this specific API calls, others handle it differnetly